config/core/test-configs.yaml

```diff
@@ -296,6 +345,33 @@ test_plans:
       job_timeout: '10'
       kselftest_collections: "lkdtm"
 
+  kselftest-membarrier:
+    <<: *kselftest
+    params:
+      job_timeout: '10'
+      kselftest_collections: "membarrier"
+    filters: *kselftest_no_fragment
+
+  kselftest-mincore:
+    <<: *kselftest
+    params:
+      job_timeout: '10'
+      kselftest_collections: "mincore"
+    filters: *kselftest_no_fragment
+
+  kselftest-net:
+    <<: *kselftest
+    params:
+      job_timeout: '10'
+      kselftest_collections: "net"
```

Comment:
I think we might not be able to run this one due to the debug network interface causing issues with kernel-side DHCP.  See this commit https://github.com/kernelci/kernelci-core/commit/ddedee95b11175e9e95941523d0872ce78c0392b

Reply:
I have a plan for bringing up an environment for that using virtualization which would avoid that issue. In any case, this is just adding the fragment and not enabling running it on any platform.

Reply:
Sure, but I'm not convinced of the value of adding dead code or config data especially when we know it can't be used.

Reply:
My goal here is to get the basic stanzas in for the kselftests since I've been finding they're a frequent source of conflicts when enabling things - for things like this where the test clearly exists upstream and is unlikely to go away it seems reasonable to have the stanza there ready for people to build on.